code/game/gamemodes/cult/runes.dm

```diff
@@ -425,6 +425,9 @@ structure_check() searches for nearby cultist structures required for the invoca
 		stone.invisibility = INVISIBILITY_MAXIMUM // So it's not picked up during transfer_soul()
 		stone.transfer_soul("FORCE", offering, user) // If it cannot be added
 		stone.invisibility = 0
+		var/put_in_hands = user.put_in_any_hand_if_possible(stone)
+		if(put_in_hands)
+			to_chat(user, "<span class='cultitalic'>Shiny dark red shard appears in your hand - your new ally.</span>")
```

Comment:
```suggestion
			to_chat(user, "<span class='cultitalic'>A glowing blood red shard appears in your hand - your new ally.</span>")
```

Reply:
Id rather keep bloodred especially for syndicate and use another term with cult

Reply:
yeah i changed it to crimson, did it not update?